afids_utils/ext/json.py

```diff
@@ -0,0 +1,154 @@
+"""Methods for handling .json files associated with AFIDS"""
+from __future__ import annotations
+
+import json
+from importlib import resources
+from os import PathLike
+
+from afids_utils.afids import AfidPosition, AfidSet
+from afids_utils.exceptions import InvalidFileError
+
+
+class ControlPoint:
+    id: str
+    label: str
+    description: str
+    associatedNodeID: str
+    position: list[float]
+    orientation: list[float]
+    selected: bool
+    locked: bool
+    visibility: bool
+    positionStatus: str
+
+
+def _get_metadata(
+    in_json: dict[str, bool | float | str | list[float]]
+) -> tuple[str, str]:
+    """Internal function to extract metadata from json files
+
+    Note: Slicer version is not currently included in the json file
```

Comment:
Note: This is perhaps another argument for dropping `slicer_version` from `AfidSet`

Reply:
Yes, I agree - if we end up dropping this, I would prefer to do it in a separate PR